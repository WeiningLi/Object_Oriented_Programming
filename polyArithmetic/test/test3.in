create a -0/1 9 1/1 8 2/8 6 3/5 5 -2/5 2 1/2 1 3/7 0
print a 
create b 1/5 1 3/12 0
print b
create c 0/1 0
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
multiply a b
multiply b c
multiply b a
multiply c a
divide c a
divide a b
divide b a

