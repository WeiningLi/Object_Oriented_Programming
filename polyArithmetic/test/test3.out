(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(1/5)x + (1/4)
0
(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (3/10)x + (5/28)
(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (7/10)x + (19/28)
(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(1)x^8 + (1/4)x^6 + (3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(-1)x^8 + (-1/4)x^6 + (-3/5)x^5 + (2/5)x^2 + (-1/2)x + (-3/7)
(1/5)x^9 + (1/4)x^8 + (1/20)x^7 + (73/400)x^6 + (3/20)x^5 + (-2/25)x^3 + (59/280)x + (3/28)
0
(1/5)x^9 + (1/4)x^8 + (1/20)x^7 + (73/400)x^6 + (3/20)x^5 + (-2/25)x^3 + (59/280)x + (3/28)
0
Quotient: 0
Remainder: 0
Quotient: (5)x^7 + (-25/4)x^6 + (145/16)x^5 + (-533/64)x^4 + (2665/256)x^3 + (-13325/1024)x^2 + (58433/4096)x + (-251205/16384)
Remainder: (1955043/458752)
Quotient: 0
Remainder: (1/5)x + (1/4)
